public/js/controllers/StatsController.js

```diff
@@ -783,6 +832,18 @@ angular.module('BlocksApp').controller('StatsController', function($stateParams,
                 .attr("class", "y axis")
                 .call(yAxis);
 
+            var tip = d3.tip()
+                .attr('class', 'd3-tip')
+                .offset([-10, 0])
+                .direction('s')
+                .html(function(d) {

```

Comment:
This is a good example of when `d` function parameter is being used inside the function, as it's being used in line 843 in `d.unixtime`

Reply:
thank you!